CONTRIBUTING.md

```diff
@@ -1,14 +1,27 @@
 # Contributing
+
 ## How to test `django-admin startproject`
-If you made changes to this boilerplate and want to test them, commit your changes and use `git archive -o boilerplate.zip HEAD` to create the template zip. Then, do a `cd ..` and a `django-admin startproject theprojectname --extension py,yml,json --name Procfile,README.md,.env.example --template=django-react-boilerplate/boilerplate.zip` to test the project bootstrap.
+
+If you made changes to this boilerplate and want to test them, do as following:
+
+- Commit your changes
```

Comment:
Should we add a parenthesis here enforcing "make sure you've installed the pre-commit"?

Reply:
@vanessa

Reply:
Not sure if making sure pre-commit is installed should be relevant in this part of readme, because it is more related with setting up the project in a dev environment than actually testing it. @lucabezerra

Reply:
@joaopslins I get your point, but my thought here was: I just created a PR for an issue listed here (to add a simple configuration line to the `base.py` file). When I was done with it, I committed and when I was about to push, I remembered we actually have a pre-commit set up, which didn't run (because I hadn't installed it). Since installing the pre-commit is not mandatory for creating a PR, my take on this is that we should reinforce that in here. What do you think?

Reply:
Agreed @lucabezerra . Please fix @vanessa